pkg/manager/member/template.go

```diff
@@ -160,14 +160,14 @@ then
 echo "waiting for pd cluster ready timeout" >&2
 exit 1
 fi
-
+{{ if eq .CheckDomainScript ""}}
 if nslookup ${domain} 2>/dev/null
 then
 echo "nslookup domain ${domain}.svc success"
 break
 else
 echo "nslookup domain ${domain} failed" >&2
-fi
+fi {{- else}}{{.CheckDomainScript}}{{end}}
```

Comment:
Will the extra space after `fi` cause a rolling update?

Reply:
no.  note  `{{-` with '-' it will remove space before it